app/components/navigation_header/header.tsx

```diff
@@ -32,7 +32,6 @@ type Props = {
     onTitlePress?: () => void;
     rightButtons?: HeaderRightButton[];
     scrollValue?: Animated.SharedValue<number>;
-    lockValue?: Animated.SharedValue<number | null>;
```

Comment:
The only place lockValue is being used is in the containerAnimatedStyle, and `lockValue` should not be part of useAnimatedStyle dependency, since it's not being used. If lockValue changed, would it cause defaultHeight and insets.top to change?